cernopendata/templates/cernopendata_theme/footer.html

```diff
@@ -98,12 +98,13 @@
       </div>
       <div class="six wide column stretched align-items-end">
         <div class="auto-column-grid">
-          <div >Powered by <a href="https://inveniosoftware.org/">Invenio</a></div>
+          <div ><a href="https://inveniosoftware.org/">Powered by Invenio</a></div>
           <div  >
             <a href="http://www.cern.ch">
               <img
                 src="{{url_for('static', filename='assets/img/cern.svg')}}"
                 width="80px"
+                alt="cern"
```

Comment:
Not sure if it makes a difference, but I'd write it in uppercase.

```suggestion
                alt="CERN"
```